BIRMINGHAM | ITP-MAY-25|Emin Akturk | Sprint 1

eminakturk opened the pull request:
## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/contributing/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

I have commited new changes as CJ requested and fixed the related problems.

cjyuan commented on Sprint-1/3-mandatory-interpret/1-percentage-change.js:
```diff
@@ -20,3 +20,17 @@ console.log(`The percentage change is ${percentageChange}`);
+//  `carPrice.replaceAll(",", "")` on line 5
+//  `priceAfterOneYear.replaceAll(",", "")` on line 6
+//  `Number(...)` on lines 5 and 6
+// b) The error occurs on line 5 there's a missing comma.
```
A comma is missing between "," and "" in the function call.
Can you find out the programming term for "," and "" so that we can more accurately describe the reason as
"a comma is missing between the ___________."

cjyuan commented on Sprint-1/3-mandatory-interpret/3-to-pounds.js:
```diff
@@ -25,3 +25,13 @@ console.log(`£${pounds}.${pence}`);
+
+// 4. const pounds = paddedPenceNumberString.substring(0, paddedPenceNumberString.length - 2): This line takes out the pounds part of the string.
+
+// 5. const pence = paddedPenceNumberString.substring(paddedPenceNumberString.length - 2).padEnd(2, "0"): This line takes out pence part and pads it to 2 characters.
```
Could we expect this program to work as intended for any valid `penceString` if we deleted `.padEnd(2, "0")` from the code?
In other words, do we really need `.padEnd(2, "0")` in this script?

cjyuan replied:
You can also just answer the question here.

eminakturk commented:
Hello CJ!

Thank you for the review. I have fixed every error you mentioned.
I'm aware of sprint 3 needs to be done which I will be this week.

Also, I have fixed Pr template as you wished and ticked the checkboxes.

Thank you.

cjyuan commented:
Changes look good.

Leaving responses directly in the comment threads makes tracking the discussion easier. You can try the approach in future PRs.

![image](https://github.com/user-attachments/assets/c841b780-5333-4f08-a568-93d94b3f4707)

Here is a simplified version of best practices ChatGPT suggested for responding to inline comments in a pull request:
> - ✅ Reply to every comment – Let the reviewer know you saw it.
> - ✏️ Make the change if needed – Fix the code if the comment points out a real issue.
> - 🤔 Explain if you don't agree – If you think the code is fine, politely explain why.
> - ✅ Mark as resolved when done – Only mark comments resolved after you fix or respond.
> - 💬 Keep replies short and polite – Be respectful and to the point.
> - ⏱️ Respond soon – Don’t wait too long to reply.
> - 🧪 Test your changes – Make sure your fixes actually work.
> - 📍 Reply directly under the comment – This keeps the conversation easy to follow.
